Assets/MixedRealityToolkit.Providers/WindowsMixedReality/XRSDK/WindowsMixedRealityDeviceManager.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.XRSDK.Input;
+using System;
+using UnityEngine;
+using UnityEngine.XR;
+using Microsoft.MixedReality.Toolkit.Windows.Utilities;
```

Comment:
Super nit: Bring this up with the other Microsoft.* stuff (alphabetical order stuff)

Reply:
Updated!